docs/source/markdown/podman-create.1.md

```diff
@@ -515,6 +515,11 @@ Not implemented
 
 Logging driver for the container. Currently available options are *k8s-file*, *journald*, *none* and *passthrough*, with *json-file* aliased to *k8s-file* for scripting compatibility.
 
+The podman info command below, will display the default log-driver for the system.
```

Comment:
Comma is ungrammatical. I won't block merge because of it, but I can't lgtm either, sorry